<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>407</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Salarios</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos/Recursos.qrc">
    <normaloff>:/imagenes/salario.png</normaloff>:/imagenes/salario.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(245, 248, 255);
selection-background-color: rgb(211, 214, 220);
selection-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Nombre: </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="inNombre"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Horas: </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="inHoras"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Jornada</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="inMatutino">
            <property name="text">
             <string>Matutino</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="inVespertino">
            <property name="text">
             <string>Vespertino</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="inNocturno">
            <property name="text">
             <string>Nocturno</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>263</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cmdCalcular">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(6, 135, 255);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>&amp;Calcular</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>281</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="outDatos">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_archivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="action_Nuevo"/>
    <addaction name="actionA_brir"/>
    <addaction name="action_Guardar"/>
    <addaction name="action_Calcular"/>
    <addaction name="action_Salir"/>
   </widget>
   <widget class="QMenu" name="menuAy_uda">
    <property name="title">
     <string>Ay&amp;uda</string>
    </property>
    <addaction name="action_AcercaDe"/>
   </widget>
   <addaction name="menu_archivo"/>
   <addaction name="menuAy_uda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_AcercaDe">
   <property name="icon">
    <iconset resource="Recursos/Recursos.qrc">
     <normaloff>:/imagenes/informacion.png</normaloff>:/imagenes/informacion.png</iconset>
   </property>
   <property name="text">
    <string>Ac&amp;erca de</string>
   </property>
  </action>
  <action name="action_Nuevo">
   <property name="icon">
    <iconset resource="Recursos/Recursos.qrc">
     <normaloff>:/imagenes/hoja-en-blanco.png</normaloff>:/imagenes/hoja-en-blanco.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionA_brir">
   <property name="icon">
    <iconset resource="Recursos/Recursos.qrc">
     <normaloff>:/imagenes/carpeta.png</normaloff>:/imagenes/carpeta.png</iconset>
   </property>
   <property name="text">
    <string>A&amp;brir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Guardar">
   <property name="icon">
    <iconset resource="Recursos/Recursos.qrc">
     <normaloff>:/imagenes/disquete.png</normaloff>:/imagenes/disquete.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Calcular">
   <property name="icon">
    <iconset resource="Recursos/Recursos.qrc">
     <normaloff>:/imagenes/calculadora.png</normaloff>:/imagenes/calculadora.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Calcular</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_Salir">
   <property name="icon">
    <iconset resource="Recursos/Recursos.qrc">
     <normaloff>:/imagenes/boton-x.png</normaloff>:/imagenes/boton-x.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>inNombre</tabstop>
  <tabstop>inHoras</tabstop>
  <tabstop>inMatutino</tabstop>
  <tabstop>inVespertino</tabstop>
  <tabstop>inNocturno</tabstop>
  <tabstop>cmdCalcular</tabstop>
  <tabstop>outDatos</tabstop>
 </tabstops>
 <resources>
  <include location="Recursos/Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
